pkg/steps/template.go

```diff
@@ -24,13 +24,22 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
 
+	"k8s.io/test-infra/prow/pod-utils/decorate"
+	"k8s.io/test-infra/prow/pod-utils/downwardapi"
+
 	templateapi "github.com/openshift/api/template/v1"
 	templateclientset "github.com/openshift/client-go/template/clientset/versioned/typed/template/v1"
 
 	"github.com/openshift/ci-tools/pkg/api"
 	"github.com/openshift/ci-tools/pkg/junit"
 )
 
+const (
+	RepoOwnerVar       = "REPO_OWNER"
+	RepoNameVar        = "REPO_NAME"
+	RepoOwnerNameLabel = "template.openshift.io/template-repo-owner-name"
```

Comment:
Could we reuse the `prow.k8s.io/refs.{org,repo} ` label names? Or `ci.openshift.io`

Reply:
`ci.openshift.io/refs-repo-owner-name`. I can't use `.{org,repo}` in the label name because we need to get the org/repo from the value. This will helps as to use the actual values as variables in grafana.

Reply:
Do you mean having 2 labels? like `ci.openshift.io/refs.org` and `ci.openshift.io/refs.repo`?

Reply:
Yes, two labels

Reply:
addressed